### ISSUE TITLE
Maps not loading?

### ISSUE THREAD

**Issue by gg2001** [PandaWhisperer/meteor-yelp-clone#3]
I have cloned this repo, and ran it but google maps doesn't load.
<img width="1280" alt="screenshot 2016-07-04 13 05 58" src="https://cloud.githubusercontent.com/assets/16451579/16550651/584403b2-41e8-11e6-8383-04b35eba272b.png">
<img width="1280" alt="screenshot 2016-07-04 13 04 11" src="https://cloud.githubusercontent.com/assets/16451579/16550652/5844a920-41e8-11e6-8a3d-3524b041a399.png">

**Comment by PandaWhisperer** [comment]
Yes, it says right there in the console what your problem is: you are using an invalid API key. 

You need to go to the [Google Developers Console](https://console.developers.google.com/apis) (sign up if necessary) and create a new project, then activate the Google Maps API for that project. 

Then, copy the newly created API key and place it in your `settings/development.json` by replacing the string `YOUR_GOOGLE_API_KEY_HERE`.

``` json
{
  "public": {
    "googleApiKey": "YOUR_GOOGLE_API_KEY_HERE"
  }
}
```

Restart the server if it's running so that the new settings are loaded (they only get read on startup), and maps should be working.

Please comment with results.

**Comment by gg2001** [comment]
Thanks a lot.

**Comment by PandaWhisperer** [comment]
@gg2001 does that mean this solved your issue?

**Comment by gg2001** [comment]
Yup. Feel free to close this issue.

**Comment by PandaWhisperer** [comment]
Thanks @gg2001. I'll close it once I've added these extra instructions to the tutorial. Thanks for testing them.
